fastlane_bot/helpers/routehandler.py

```diff
@@ -1358,14 +1369,13 @@ def _calc_carbon_output(
         Decimal
             The amount out.
         """
+        assert tkn_in not in self.ConfigObj.NATIVE_GAS_TOKEN_ADDRESS,"[routehandler.py _calc_carbon_output] Function does not expect native gas token as input."
```

Comment:
Let's start get rid of these `in` and `not in`, shall we?

Using `!=` here would do the job fine AND be more readable (and probably also faster).

Reply:
Changed to !=